pithos/pithos.py

```diff
@@ -1039,6 +1068,7 @@ def on_prefs_response(self, widget, response):
 
         email = self.settings.get_string('email')
         if response == Gtk.ResponseType.APPLY:
+            self.worker_run(self.on_explicit_content_filter_checkbox)
```

Comment:
I know that's why I was using GLib.idle_add and not the worker but you told me to use the worker?